metrics/src/main/java/com/lightstep/tracer/metrics/Sender.java

```diff
@@ -37,6 +37,7 @@ final O exec(final long timeout) throws Exception {
     // We might not want the first report, as we'd get accumulated data.
     if (!readyToReport) {
       readyToReport = true;
+      setRequest(null);
```

Comment:
we skip first metric report (duration for it is 0), 
but request data still in a queue to send in next iteration,
therefore we need to drop unwanted request data.